lib/metrics/StatsClient.js

```diff
@@ -105,6 +129,37 @@ class StatsClient {
         return this._redis.incrEx(key, this._expiry, callback);
     }
 
+    /**
+     * Handles getting a list of global keys.
+     * @param {array} ids - Service identifiers
+     * @param {object} log - Werelogs request logger
+     * @param {function} cb - Callback
+     * @return {undefined}
+     */
+    getAllGlobalStats(ids, log, cb) {
```

Comment:
Just a suggestion, but maybe move this to StatsModel to separate out methods that will be used by utapi and methods that will be used by backbeat

Reply:
Okay, will move this. I didn't realize there were plans to have another client for Utapi.

Reply:
sorry bad wording on my part. I meant StatsClient is used by utapi, and StatsModel was adjusted to fit for backbeat metrics. Since this method will be only used for backbeat, I thought it'd be nice to keep it in StatsModel